### PR TITLE
chore: increase pg connections

### PR DESCRIPTION
**Pull request by chris13524**
# Description

10 is low, we have a lot more available connections we can leverage. If these connections are used up we get a server error to better to have more.

## How Has This Been Tested?

Automated tests

## Due Diligence

* [ ] Breaking change
* [ ] Requires a documentation update
* [ ] Requires a e2e/integration test update

### CODE REVIEW

File: src/lib.rs [hunk 1]
```diff
@@ -51,7 +51,10 @@ pub async fn bootstrap(mut shutdown: broadcast::Receiver<()>, config: Configurat
 
     let analytics = analytics::initialize(&config, s3_client, geoip_resolver.clone()).await?;
 
-    let postgres = PgPoolOptions::new().connect(&config.postgres_url).await?;
+    let postgres = PgPoolOptions::new()
+        // https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/aurora-serverless-v2.setting-capacity.html#aurora-serverless-v2.max-connections
+        .max_connections(100)
```
Comment (xav) [inline src/lib.rs:56]: can't we have this as a config value defined from TF?

**Review by xav — CHANGES_REQUESTED** [review]

File: src/lib.rs (hunk 1 above)
Comment (chris13524) [reply src/lib.rs]: Added in 453997a

**Review by xav — APPROVED** [review]
Not a blocker, but the repo is set up to use [terraform-docs](https://terraform-docs.io/) that will update all the READMEs for you from the TF code (you just need to run `terraform-docs .` in the `/terraform` directory). It's good hygiene to run it when you modify the infra.

**Comment by chris13524** [comment]
@xav I don't think terraform-docs is working for me. I ran it, but it only modified something and didn't add my new input at all:

image

Would like to run it automatically, but we can figure it out in another PR.

**Pull request merged**